docs/Performance.md

```diff
@@ -86,6 +86,19 @@ but their receipt is not necessary for the scroll to occur).
 
 When running a bundled app, these statements can cause a big bottleneck in the JavaScript thread. This includes calls from debugging libraries such as [redux-logger](https://github.com/evgenyrodionov/redux-logger), so make sure to remove them before bundling.
 
+> There is a [babel plugin](https://babeljs.io/docs/plugins/transform-remove-console/) can remove all console.* calls. You need to install it first using `npm install babel-plugin-transform-remove-console --save`, and then edit (or create) `.babelrc` under your project directory like the following:
```

Comment:
You can probably put console.* in code format too using the backticks. e.g., `console.*`